docs/index.bs

```diff
@@ -3582,8 +3602,9 @@ spec: rfc7231; urlPrefix: https://tools.ietf.org/html/rfc7231
     <ol>
       <li><a>Assert</a>: a <a>Record</a> with the \[[value]] equals to <var>registration</var> is contained in <a href="#dfn-scope-to-registration-map">scope to registration map</a>.</li>
       <li><a>Assert</a>: <var>registration</var>'s <a href="#dfn-active-worker">active worker</a> is not null.</li>
+      <li>Let <var>event</var> be the <a for="/">functional event</a> for which this algorithm was invoked.</li>
```

Comment:
Added an argument, event. Please update the notifications API accordingly when it's landed.